sunpy/net/dataretriever/sources/gong.py

```diff
@@ -150,9 +159,7 @@ def download_from_nso(id, ObsID, time_range):
             try:
                 check = urllib2.urlopen(urls)
                 final_result.append(urls)
-            except urllib2.HTTPError,e:
-                pass
-            except urllib2.URLError,e:
+            except:
```

Comment:
I don't think I've asked for passing all the errors... but combining both into one.